gulpfile.mjs

```diff
@@ -571,7 +574,10 @@ function createSandboxBundle(defines, extraOptions = undefined) {
 
 function createWorkerBundle(defines) {
   const workerFileConfig = createWebpackConfig(defines, {
-    filename: defines.MINIFIED ? "pdf.worker.min.mjs" : "pdf.worker.mjs",
+    filename:
+      defines.MINIFIED && !defines.MOZCENTRAL && !defines.GECKOVIEW
+        ? "pdf.worker.min.mjs"
+        : "pdf.worker.mjs",
```

Comment:
Similarly, can we change the following instead?
https://github.com/mozilla/pdf.js/blob/b070a502c33aa08788e31e8f2afb675df1405374/web/app_options.js#L515